DB/database.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import List
+from typing import Optional
+from datetime import datetime
+from uuid import UUID
+# from email_validate import validate
+from macros import MacrosStatus
+
+from sqlalchemy import String, ForeignKey, create_engine
+from sqlalchemy.orm import DeclarativeBase, \
+    validates, mapped_column, Mapped, relationship, session
+
+par_dir = os.path.dirname(os.path.abspath(__file__))
+db_path = os.path.join(par_dir, "db.sqlite3")
+
+
+# Create the base model class
+class Base(DeclarativeBase):
+    pass
+
+
+class User(Base):
+    __tablename__ = 'users'
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    email: Mapped[str] = mapped_column(String(255), nullable=False, unique=True)
+    uploads: Mapped[List["Upload"]] = relationship(back_populates="user",
+                                                   cascade="all, delete-orphan")
+
+
+class Upload(Base):
```

Comment:
Instead of all this empty lines, you can add properties and functions like is_finished or processing_time